fala/apps/adviser/forms.py

```diff
@@ -16,6 +18,54 @@
     ("Solicitor", "Solicitor"),
 ]
 
+Region = Enum(
```

Comment:
Just thinking, can we break these up into their own Python files and import there usage instead of all being in the `forms.py` file.

Reply:
We have moved this to it's own file as it was used by both views and forms